server/src/test/java/com/hedera/block/server/config/ServerMappedConfigSourceInitializerTest.java

```diff
@@ -71,35 +93,46 @@ class ServerMappedConfigSourceInitializerTest {
-            fail("Class %s is missing the ConfigData annotation! All config classes MUST have that annotation present!"
-                    .formatted(config.getSimpleName()));
+            fail(
+                    "Class [%s] is missing the ConfigData annotation! All config classes MUST have that annotation present!"
+                            .formatted(config.getSimpleName()));
         } else {
             final ConfigData configDataAnnotation = config.getDeclaredAnnotation(ConfigData.class);
             final String prefix = configDataAnnotation.value();
-            for (final RecordComponent recordComponent : config.getRecordComponents()) {
+            final RecordComponent[] fieldsToVerify = Arrays.stream(config.getRecordComponents())
+                    .filter(recordComponent -> !fieldNamesToExclude.contains(recordComponent.getName()))
+                    .toArray(RecordComponent[]::new);
+            LOGGER.log(
+                    Level.INFO,
+                    "Checking fields %s\nfor config class [%s]."
+                            .formatted(Arrays.toString(fieldsToVerify), config.getSimpleName()));
+            for (final RecordComponent recordComponent : fieldsToVerify) {
+                final String fieldName = recordComponent.getName();
                 if (!recordComponent.isAnnotationPresent(ConfigProperty.class)) {
                     fail(
-                            "Field %s in %s is missing the ConfigProperty annotation! All fields in config classes MUST have that annotation present!"
-                                    .formatted(recordComponent.getName(), config.getSimpleName()));
+                            "Field [%s] in [%s] is missing the ConfigProperty annotation! All fields in config classes MUST have that annotation present!"
+                                    .formatted(fieldName, config.getSimpleName()));
                 } else {
-                    final String expectedMappedName = "%s.%s".formatted(prefix, recordComponent.getName());
+                    final String expectedMappedName = "%s.%s".formatted(prefix, fieldName);
                     final Optional<ConfigMapping> matchingMapping = Arrays.stream(SUPPORTED_MAPPINGS)
                             .filter(mapping -> mapping.mappedName().equals(expectedMappedName))
                             .findFirst();
                     assertThat(matchingMapping)
                             .isNotNull()
                             .withFailMessage(
                                     "Field [%s] in [%s] is not present in the environment variable mappings! Expected config key [%s] to be present and to be mapped to [%s]",
-                                    recordComponent.getName(),
+                                    fieldName,
                                     config.getSimpleName(),
                                     expectedMappedName,
                                     transformToEnvVarConvention(expectedMappedName))
```

Comment:
nit: This could be extracted in a variable with descriptive name.

Reply:
✅ Done. Thanks, I've missed that. Updated.